ci/pipeline.yml

```diff
@@ -160,11 +162,23 @@ resources:
     uri: https://github.com/18F/cg-secret-rotation
     branch: {{secret-rotation-git-branch}}
 
+- name: secret-rotation-config-development
+  type: git
+  source:
+    uri: https://github.com/18F/cg-secret-rotation
+    branch: {{secret-rotation-git-branch-development}}
+
 - name: bosh-config
   type: git
   source:
     uri: https://github.com/18F/cg-deploy-bosh
-    branch: master
+    branch: {{bosh-git-branch}}
```

Comment:
Why are we changing this to be configurable again?

Reply:
Because secrets generation lives in `cg-deploy-bosh` this makes testing integration w/ changes like https://github.com/18F/cg-deploy-bosh/pull/178 easier... Not sure how much value that really add though vs. hand propping in dev.